tools/perfscale-load-generator/README.md

```diff
@@ -0,0 +1,79 @@
+# Perscale Load Generator
+
+The load generator is a tool aimed at stressing the Kubernetes and KubeVirt control plane by creating several objects (e.g., VM, VMI, and VMIReplicaSet). The main functionality it provides can be summarized as follows:
+- Create the objects declared in a workload description.
+- Create VMIs in one namespace or one VMI per namespace.
+- Wait VMIs be created in each iteration.
```

Comment:
/s/Wait VMIs/Wait for VMIs to ;)

Reply:
@marceloamaral can you address?  I think your PR is pretty much ready to merge